### PR TITLE
Improving build process and development workflow 

### LINKED ISSUE #74: Use Grunt to run a connect server on a watch with live-reload for dev purposes

### LINKED ISSUE #75: Need automated unit test build tasks for the js scripts and automated jshint task run

### PR DESCRIPTION
**Pull request by nelsonomuto**
fixes #74 - configured a server with a watch and live-reload for easier dev workflow
fixes #75 - configured test task to run unit tests

new build and workflow commands:
- `grunt serve`: this will start a server with a watch and livereload for dev purposes.
- `grunt build`: build the code, run the css compilers based on provided target.
- `grunt jshint`: lint the js code.
- `grunt test`: run unit tests on the browser using jasmine framework.

changes to existing source include: 
- removed double quotes jshint rule, too strict. added jshint task to automated build.
- made ripples a global variable in scripts/ripples.js to pass the jshint unused var test.

### CODE REVIEW

**Comment by FezVrasta** [comment]
Thanks for the PRs... I can't get why there are two PRs though... may you explain it?

File: .jshintrc [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {
     "bitwise": true,
-    "camelcase": true,
```
Comment (FezVrasta) [inline .jshintrc:3]: would be better keep this to ensure homogeneity

File: .jshintrc [hunk 2]
```diff
@@ -15,7 +14,6 @@
     "nonbsp": true,
     "nonew": true,
     "plusplus": false,
-    "quotmark": "double",
```
Comment (FezVrasta) [inline .jshintrc:18]: same as "camelcase"

**Comment by nelsonomuto** [comment]
The first one had a few errors I needed to fix, I will add back the jshint and do another PR, I just found a small issue I need to fix on the livereload task. Bear with me, ignore this PR for now.

**Pull request closed without merging**